models/actions/variable.go

```diff
@@ -55,24 +53,25 @@ type FindVariablesOpts struct {
 	db.ListOptions
 	OwnerID int64
 	RepoID  int64
+	Name    string
 }
 
 func (opts FindVariablesOpts) ToConds() builder.Cond {
 	cond := builder.NewCond()
-	cond = cond.And(builder.Eq{"owner_id": opts.OwnerID})
-	cond = cond.And(builder.Eq{"repo_id": opts.RepoID})
+	if opts.OwnerID > 0 {
```

Comment:
I don't think this is right.

Reply:
fix in [1806ad7](https://github.com/go-gitea/gitea/pull/29520/commits/1806ad7ca2de34204e1a7b8bbc5a233c5ece16e2)